bitrise.yml

```diff
@@ -0,0 +1,122 @@
+format_version: 5
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+
+app:
+  envs:
+  - BITRISE_STEP_ID: bitrise-step-build-router-start
+  - BITRISE_STEP_VERSION: "0.9.0"
+  - BITRISE_STEP_GIT_CLONE_URL: https://github.com/bitrise-steplib/bitrise-step-build-router-start.git
+  - MY_STEPLIB_REPO_FORK_GIT_URL: $MY_STEPLIB_REPO_FORK_GIT_URL
+  - TEST_WORKFLOWS: $TEST_WORKFLOWS
+  - ACCESS_TOKEN: $ACCESS_TOKEN
+
+workflows:
+  test:
+    before_run:
+    - audit-this-step
+    - go-tests
```

Comment:
use the new go steps: https://github.com/godrei/steps-generate-changelog/blob/master/bitrise.yml#L9